atlassian_jwt_auth/signer.py

```diff
@@ -23,7 +27,7 @@ def __init__(self, issuer, private_key_retriever, **kwargs):
             raise ValueError("lifetime, '%s',exceeds the allowed 1 hour max" %
                              (self.lifetime))
 
-    def _generate_claims(self, audience, **kwargs):
+    def _generate_claims(self, audience, additional_claims=None):
```

Comment:
Being explicit is also future compatible unless you remove an argument (which is still trivially easy since this is only called from a single place); I don't think there's a difference except for readability.